hackathon_site/registration/admin.py

```diff
@@ -1,13 +1,54 @@
+import csv
+import functools
+
 from django.contrib import admin
+from django.http import HttpResponse
+
 from registration.models import Application, Team as TeamApplied
 
 
+def rgetattr(obj, attr, *args):
+    """
+    Recursive getattr, allows for nested attributes.
+    For example if you're in the Application class, you can't do getattr for user.first_name
+    But with rgetattr you can.
+    """
+
+    def _getattr(obj, attr):
+        return getattr(obj, attr, *args)
+
+    return functools.reduce(_getattr, [obj] + attr.split("."))
+
+
 class ApplicationInline(admin.TabularInline):
     model = Application
     autocomplete_fields = ("user",)
     extra = 0
 
 
+class ExportCsvMixin:
+    export_field_names = []
```

Comment:
Can you rework this so it's a list of tuples `(field attribute, field name)`? Would be a little bit easier to read, and would be inline with semantics for choice fields and dropdowns.